### PR TITLE
Make beautifyFileSize public

### PR DESCRIPTION
**Pull request by juuz0**
Requirement for https://github.com/kiwix/kiwix-desktop/pull/946

### CODE REVIEW

**Comment by kelson42** [comment]
@juuz0 Not sure this function is documented properly (in readtjedocs), please add the necessary documentation.

**Comment by mgautierfr** [comment]
> Not sure this function is documented properly (in readtjedocs), please add the necessary documentation.

It will be documented this way : https://libkiwix--971.org.readthedocs.build/en/971/api/function_tools_8h_1a18e272a0e35c09646b78f566e92ac292.html

**Comment by kelson42** [comment]
@mgautierfr actually not bad. LGTM if the number given is not expecting to be in a special unity... because this is not written in the doc.

**Comment by mgautierfr** [comment]
The name of the function is `beautifyFileSize` so yes, we expect a size and so the unit is `B`/`KB`/`MB`/`GB`

**Comment by kelson42** [comment]
@mgautierfr If the result is given with a unity, then the unity of the input expect should be specified in the documentaiton. Here I guess Bytes.

**Comment by juuz0** [comment]
@kelson42 updated

File: include/tools.h [hunk 1]
```diff
@@ -216,5 +216,14 @@ std::map<std::string, std::string> getNetworkInterfaces();
  */
 std::string getBestPublicIp();
 
+/** Converts file size to human readable format
+ * 
+ * This function will convert a number to its equivalent size using units.
+ * 
+ * @param file size in bytes
```
Comment (mgautierfr) [inline include/tools.h:223]: The first word after `@param` must be the parameter name. So:
```
@param number file size in bytes.
```

And finish the sentences (even param and return descriptions) with `.`

**Review by mgautierfr — APPROVED** [review]

**Pull request merged**